README.md

```diff
@@ -1,67 +1,232 @@
 
-# Quick start
+## Usage
+
 Depending on your OS, you can invoke `frodo-linux`, `frodo-macos` or `frodo-windows.exe` binary.
-`frodo` uses commands and sub-commands to provide the functionality stated above.
+`frodo` uses commands and sub-commands to provide the functionality stated above. All commands support the help argument as such
+
+```shell
+frodo journey -h
+```
+
+```shell
+Usage: frodo journey [options] [command]
+
+Manage journeys/trees.
+  ```
+
+### Commands
+
+- [connections](#connections)
+- [idm](#idm)
+- [info](#info)
+- [journey](#journey)
+- [logging](#logging)
 
-## Examples
-### Export a journey
+#### Connections
+
+Use this command to list, add, or delete your connections. The CLI creates connection configurations automatically when you use other commands, so you don't have to run this command to initiate a connection.
+
+##### Examples
+
+List saved connections
+
+```shell
+frodo connections list
 ```
-$ ./frodo-linux journey -h https://<forgerock-cloud-tenant>/am -u <tenant.admin@tenant.com> -p my_5up3rs3cr3tp@ssw0rd export -t Registration -r alpha -f Registration.json
+
+##### Sample output
+
+The command displays the connections found in the **`frodorc`** file, located in the user's directory.
+
+```shell
+[Host] : [Username]
+- [https://openam-ali-demo.forgeblocks.com/am] : [ali.falahi@forgerock.com]
+Any unique substring of a saved host can be used as the value for host parameter in all commands
 ```
-This will export a journey named `Registration` in `alpha` realm and save the contents in a file called `journey-Registration.json`. If the `-f <filename>` is omitted, the journey JSON data is dumped to STDOUT.
 
-### Import a journey
+The command below lists all journeys in the specified realm, but the main idea here is when you supply the host, username, and `password` frodo will automatically save the connection in the **`frodorc`** file. The `password` value will be encrypted with `AES-256`
+
+```shell
+frodo journey list https://<tenant-url>/am <realm> <username> <password>
 ```
-$ ./frodo-linux journey -h https://<forgerock-cloud-tenant>/am -u <tenant.admin@tenant.com> -p my_5up3rs3cr3tp@ssw0rd import -t ImportTest -r alpha -f journey-ImportTest.json
+
+One the connection is saved, you don't need to include the `host`, `username`, and `password` in your future commands. You'll simply use a substring in your tenant/forgerock deployment domain name to define the connection. For example, if your tenant name is `https://login.acme.forgeblocks.com/am`
+
+**`acme`** is a unique substring in your tenant url, and if it's unique across your saved connections then this enables frodo to execute the same command from above like:
+
+```shell
+frodo journey list acme
```

Comment:
Maybe I'm too picky but if we use ACME as our sample, we use it everywhere. I like that better than using one of our named tenants but all our samples should use the same tenant name.

Reply:
Yeah, I wasn't too sure if it's clear or not. But I agree we should keep it consistent. I just wanted to avoid copy/pasta